src/workerd/api/queue-test.js

```diff
@@ -15,12 +15,13 @@ export default {
     if (pathname === "/message") {
       const format = request.headers.get("X-Msg-Fmt") ?? "v8";
       if (format === "text") {
+        assert.strictEqual(request.headers.get("X-Msg-Delay-Secs"), "2")

```

Comment:
This part of this test looks a little fragile to me, but I also don't know if there's an easy way to refactor it.